api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -2448,6 +2449,22 @@ private void enableLabs() {
                 "owa/labworkflow/index.html?patient={{patient.uuid}}#/LabResults",
                 Privileges.TASK_VIEW_LABS.privilege(),
                 null));
+
+        apps.add(addToClinicianDashboardFirstColumn(app(Apps.COVID_LAB_RESULTS,
+                "pihcore.labResults.covid",
+                "fas fa-fw fa-sun",
+                "owa/orderentry/index.html?patientId={{patient.uuid}}",
```

Comment:
I don't think we want a link to order entry here.  Ideally, it would go to the Lab Workflow "lab results" page, but unfortunately that page won't currently show these results, so we likely want to just leave this blank for now, and not link anywhere.